python/python/benchmarks/test_kmeans.py

```diff
@@ -16,12 +16,11 @@
 import pytest
 from lance.util import KMeans
 
-CLUSTERS = 1024
+CLUSTERS = 32
```

Comment:
kmean is `O(n^2)` to # of clusters. Would this number too small

Reply:
> Would this number too small

What do you mean? Too small for what? Does this trigger different code paths if the values is lower?

Right now this benchmark takes far too long. I'm trying to keep these benchmarks so they can all run in just a few minutes. Most of these should just take a second or two to run. We can get plenty of samples for tracing and flamegraphs in that time. This single benchmark takes 10+ minutes on my laptop, which is annoying to wait for. We can always run one-off large scale benchmarks when we want to wait longer for the result.

Reply:
I see. it is fair.

Reply:
I've restored this test as it was, but put it behind a `"slow"` mark so it's easy to filter out.